### DISCUSSION TITLE
Solution 1.9.0 R5 (FHIR 4.4.0) doesn't compile

### DISCUSSION THREAD

**Discussion by luisgmgouveia** [FirelyTeam/firely-net-sdk#2729] [General]
Hi guys,

The following version doesn't compile:

[v1.9.0-beta-june2020-r5](https://github.com/FirelyTeam/firely-net-sdk/tree/v1.9.0-beta-june2020-r5)

I need it as I need to support FHIR 4.4.0

The problem is this (after git pull):

![image](https://github.com/FirelyTeam/firely-net-sdk/assets/57350126/617926f1-7bdf-4d02-906a-aa288efdcb4b)

Can you please help? Many thanks!

**Comment by mmsmits** [comment]
Hi @luisgmgouveia,

You should clone the "common"  submodule as well when cloning the repository using:
`git clone --recurse-submodules -j8 https://github.com/FirelyTeam/fhir-net-api.git `

which is also documented here: [Getting started](https://github.com/FirelyTeam/firely-net-sdk/tree/v1.9.0-beta-june2020-r5?tab=readme-ov-file#getting-started)
I want to add the note that this version of FHIR was a pre-release of R5 and this version of the SDK that is also no longer supported.

**Reply by luisgmgouveia** [reply]
Hi @mmsmits , thanks, that worked!

_this version of FHIR was a pre-release of R5 and this version of the SDK that is also no longer supported._

I know but my client uses FHIR 4.4.0 and isn't very keen on upgrading in the short term. I have no choice. Your first R5 release (2.0.0) runs FHIR 4.5.0, which is too recent for me. I need this one (1.9.0 R5).

Thanks!

**Reply by mmsmits** [reply]
Alright, good luck!
